src/util/update-subject-count.js

```diff
@@ -0,0 +1,34 @@
+/*------------------------------------------------------------------------------------------------\
+  Annotate number of participants based on current filters, number of participants in all, and
+  the corresponding percentage.
+  Inputs:
+    chart - a webcharts chart object
+    id_col - a column name in the raw data set (chart.raw_data) representing the observation of interest
+    id_unit - a text string to label the units in the annotation (default = 'participants')
+    selector - css selector for the annotation
+\------------------------------------------------------------------------------------------------*/
+
+export default function(chart, id_col, selector, id_unit) {
+  //count the number of unique ids in the data set
+    const totalObs = d3.set(chart.super_raw_data
```

Comment:
Why do we need super_raw_data instead of just the normal raw_data here? 

I see that convention quite a bit. Maybe we should just add it to webcharts?